components/EpisodeTools.vue

```diff
@@ -22,8 +22,6 @@ const props = defineProps({
   },
 })
 
-const emit = defineEmits(['toggleTranscript'])
```

Comment:
was anything using this emit?

Reply:
Deepsource said no, and everything seems to be working. I couldn't use the function, it had to be a link